torchvision/ops/poolers.py

```diff
@@ -1,11 +1,11 @@
-import torch
-from torch import nn, Tensor
+from typing import Dict, List, Optional, Tuple, Union
 
+import torch
 import torchvision
-from torchvision.ops import roi_align
+from torch import Tensor, nn
 from torchvision.ops.boxes import box_area
 
-from typing import Optional, List, Dict, Tuple, Union
+from .roi_align import roi_align
```

Comment:
Our current implementation relies on a specific import order in `torchvision/ops/__init__.py` since we have `torchvision/ops/roi_align.py` as well as a `roi_align` function. Thus, depending on the current stage of the imports `from torchvision.ops import roi_align` gives you either the module or the function.

In the future we should avoid name conflicts between modules and functions. For example, we could rename the module to `_roi_align.py` since all its functionality is available through `torchvision.ops`.